### PR TITLE
[FEATURE] put fluent doc "How to connect to sql data using a query" under test

### PR DESCRIPTION
**Pull request by TrangPham**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted

    ```
    black .

    ruff . --fix
    ```
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sql_data_using_a_query.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        gx.__file__,
+        "..",
+        "..",
+        "tests",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+        "sqlite",
+        "yellow_tripdata.db",
+    ).resolve(strict=True)
+)
```
Comment (Kilo59) [inline tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sql_data_using_a_query.py]: Don't need the string conversion here.

```suggestion
sqlite_database_path = pathlib.Path(
        gx.__file__,
        "..",
        "..",
        "tests",
        "test_sets",
        "taxi_yellow_tripdata_samples",
        "sqlite",
        "yellow_tripdata.db",
    ).resolve(strict=True)
```

File: tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sql_data_using_a_query.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+my_asset = datasource.get_asset("my_asset")
+assert my_asset
+
+my_batch_request = my_asset.build_batch_request()
+batches = my_asset.get_batch_list_from_batch_request(my_batch_request)
+assert len(batches) == 1
+assert set(batches[0].columns()) == {
+    "passenger_count",
+    "total_amount",
+}
```
Comment (alexsherstinsky) [inline tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sql_data_using_a_query.py:55]: @TrangPham I am surprised that there are only two columns (I thought that there were 18 columns).  Is the dataset different?  Thanks.

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Small question -- thanks!

File: tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sql_data_using_a_query.py (hunk 2 above)
Comment (TrangPham) [reply tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sql_data_using_a_query.py:55]: This is for a dataset from a sql query. The sql query only selects these two columns.

**Review by alexsherstinsky — APPROVED** [review]
LGTM -- thank you!

**Review by Shinnnyshinshin — APPROVED** [review]
LGTM to me as well. Thank you @TrangPham

**Pull request merged**